makeflow/src/makeflow.c

```diff
@@ -201,7 +201,7 @@ static void makeflow_node_force_rerun(struct itable *rerun_table, struct dag *d,
 Decide whether to rerun a node based on batch and file system status.
 */
 
-void makeflow_node_decide_rerun(struct itable *rerun_table, struct dag *d, struct dag_node *n)
+void makeflow_node_decide_rerun(struct itable *rerun_table, struct dag *d, struct dag_node *n, int silent)
```

Comment:
It looks good.

Please add a comment to explain why we added the silent argument (i.e., how it relates to clean_mode).